### PR TITLE
Remove mention of reduce() in decimal.Context.normalize().

### PR DESCRIPTION
**Pull request by sir-sigurd**
Context.reduce() exists in cdecimal, but was not merged in cPython.

### CODE REVIEW

**Comment by vstinner** [comment]
This change makes the docstring consistent with Doc/library/decimal.rst:
"normalize(x): Reduces x to its simplest form."
https://docs.python.org/dev/library/decimal.html#decimal.Context.normalize

**Review by vstinner — APPROVED** [review]
LGTM.

**Comment by skrah** [comment]
`Reduce` refers to the fact that `normalize` is called `reduce` in the latest version of the specification:

   http://speleotrove.com/decimal/daops.html#refredu

It is mentioned so that people know it's an alias for the official `reduce`.

**Comment by vstinner** [comment]
> Reduce refers to the fact that normalize is called reduce in the latest version of the specification: (...)

If it's an useful information, it should also be in the .rst documentation, no?
https://docs.python.org/dev/library/decimal.html#decimal.Context.normalize

"Alias for reduce(x)." is commonly used in Python when reduce() does exist. Maybe rephrase it as something like: "Function called reduce(x) in Decimal Arithmetic Specification."?

**Comment by sir-sigurd** [comment]
> `Reduce` refers to the fact that `normalize` is called `reduce` in the latest version of the specification:

IMHO this leads only to confusion since the specification is not referred anywhere in docstrings of `Context`. Also if it's considered to be an useful information, why `reduce` is not mentioned in docstring of `Decimal.normalize()` the same way it's mentioned in `cdecimal.Decimal.normalize()`?

**Comment by skrah** [comment]
On Tue, Nov 06, 2018 at 07:08:38PM -0800, Sergey Fedoseev wrote:
> Also if it's considered to be an useful information, why `reduce` is not mentioned in docstring of `Decimal.normalize()` the same way it's mentioned in `cdecimal.Decimal.normalize()`?

I would think that's because normalize() in _pydecimal predates the latest
version of the specification.

Feel free to open a bugs.python.org issue that clarifies the reduce()/normalize()
relationship.
